torch/csrc/jit/mobile/profiler_edge.cpp

```diff
@@ -27,17 +26,26 @@ KinetoEdgeCPUProfiler::KinetoEdgeCPUProfiler(
   if (with_modules || with_stack) {
     auto post_processing = [this, with_stack, with_modules](
                                std::vector<profiler::KinetoEvent>& events) {
+      std::string no_debug_info("Model was not saved with debug information");
```

Comment:
Yeah I think this is needed for now (until we decide to always or never use debug handles).

Reply:
I think we would need this, as you pointed out, for embedded or other deployment scenario where we explicitly dont want any overhead.